src/oc/web/actions.cljs

```diff
@@ -59,10 +60,10 @@
       (if-let [board-data (first (filter #(= (:slug %) (router/current-board-slug)) boards))]
         ; Load the board data since there is a link to the board in the org data
         (when-let [board-link (utils/link-for (:links board-data) ["item" "self"] "GET")]
-          (api/get-board board-link))
-        ; The board wasn't found, showing a 404 page
+          (sa/section-get board-link))
+
         (if (= (router/current-board-slug) utils/default-drafts-board-slug)
-          (utils/after 100 #(dispatcher/dispatch! [:board utils/default-drafts-board]))
+          (utils/after 100 #(sa/section-get utils/default-drafts-board))
```

Comment:
@thepug this needs to dispatch the board data without loading it. It's needed to make sure that the user doesn't end up to a 404 when visiting the drafts board if it's empty.
The storage doesn't include a drafts board in the org boards list if you don't have any but can be that the user force the url to it, in this case i forge the draft board data (check utils/default-drafts-board)